client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -79,7 +79,18 @@ export const getTask = (
 		case CHECKLIST_KNOWN_TASKS.EMAIL_VERIFIED:
 			taskData = {
 				timing: 1,
-				title: translate( 'Confirm your email address' ),
+				title: translate(
+					'Confirm your email address {{emailWrapper}} %(email)s {{/emailWrapper}} ',
```

Comment:
We seem to have a few extra leading and trailing spaces here and there that could be removed:

```suggestion
					'Confirm your email address {{emailWrapper}}%(email)s{{/emailWrapper}}',
```